common/djangoapps/microsite_configuration/backends/database.py

```diff
@@ -126,6 +127,8 @@ class DatabaseMicrositeTemplateBackend(BaseMicrositeTemplateBackend):
     """
     Specialized class to pull templates from the database
     """
+    def get_template_path(self, relative_path, **kwargs):
+        return relative_path
```

Comment:
The problem with this using the base class is that if a template exists in the microsite directory, then get_template_path returns something like `'testsite/template/my-template.html'` and `get_template()` expects `'my-template.html'`, therefore it finds nothing and the database template is not returned. Since this is the DatabaseTemplateBackend I figured we would want to avoid that behavior